superset/views/core.py

```diff
@@ -734,7 +734,7 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements,too-m
 
             # if feature enabled, run some health check rules for sqla datasource
             if hasattr(datasource, "health_check"):
-                datasource.health_check()
+                datasource.health_check_message = datasource.health_check()
```

Comment:
Does this override the default `@property` getter or just update the cache?

Reply:
@ktmud good call. Lines 735–737 can actually be removed.